ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/Errors.java

```diff
@@ -63,6 +69,23 @@
   public static final int ERROR_CODE_SERVER_ERROR =
       toErrorCode(INTERNAL_SERVER_ERROR.code());
 
+  /*
```

Comment:
This looks like it was meant to be removed.

Reply:
ack